src/components/PriceBar/TileColorBoard.tsx

```diff
@@ -9,37 +9,50 @@ const TileColorBoard: React.FC = () => {
   const tileBlocks = useStoreSelector((state) => state.priceBar.blocks);
   const courts = useStoreSelector((state) => state.courtSize);
   const { data } = useGetPriceQuery(0);
-  const [useTotalPrice, setTotalPrice] = useState<string>("0.00");
+  const priceList = data?.find((item: IPriceCalculation) => !item.isDeleted);
+  const [useTotalPrice, setTotalPrice] = useState<string>("Loading...");
```

Comment:
It is not acceptable to set a "loading" string into an apparent number variable. Please divide them into 2 variables and use conditions to render the loading text or price number.

Reply:
actually, the final calculated result will be converted to string type to adjust the price format "X.X0"